WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2328,6 +2340,45 @@ public void onMediaUploadCancelClicked(String localMediaId) {
     }
 
     @Override
+    public void onMediaDeleted(String localMediaId) {
+        if (!TextUtils.isEmpty(localMediaId)) {
```

Comment:
addressed this one in a765480, but would probably be good to open a different PR to only take care of this in particular (handling of mediaIds throughout the app)